### ISSUE TITLE
SEO Google

### ISSUE THREAD

**Issue by maxliqd** [liqd/digitalstrategie#620]
**Essential Information** German preview on google contains English text from alt text header image and first content block. 
**URL:**
**device & browser:**
**operating system (incl. version):**
**behaviour:**
**expected behaviour:** It should be in German at least, ideally the content should be taken from the meta tags. Browsers set to English should ideally get English content in the google preview. 

**Additional information if relevant**
**user role:**
**important screensize:**
**Comment/Question:**
Do we need a SEO story for this?

**Screenshot if design related**:
Since issues are public, make sure to not display personal or secret data.
![image](https://user-images.githubusercontent.com/94360651/221563839-b34ca1eb-b2da-4dda-8e8d-9f35bbcf9efe.png)

**Comment by fuzzylogic2000** [comment]
This might not work as we don't know if people are using google in English or German and our translations might just not work with the meta tags.
We will check if the right things from the meta tags are displayed, though.

**Comment by philli-m** [comment]
I found this site https://seositecheckup.com/ and it looks like it is now working on dev but it also seems to work on prod but reading up about it, it seems like Google can decide what it shows there, as we provide the information in English, it's possible that google chooses that over German, to make it weirder I get German description on chrome and english description on firefox, both my browsers are set to english?!

**Comment by maxliqd** [comment]
@philli-m Checked dev and prod on another test site (https://www.seobility.net/de/seocheck/) again. Looked ok to me too. I don't quite know how we can stop google choosing that English text from the teaser image block. My suggestion would be to exchange the content after the prod release, maybe try another block for that text. Don't see what else we could do.

**Comment by philli-m** [comment]
what do you mean when you say 'maybe try another block for that text'? do you want to add an extra field?

**Comment by maxliqd** [comment]
@philli-m No, I just meant I would put the content in another teaser. We need to adapt the structure of the start page anyway.

**Comment by philli-m** [comment]
Agreed, we have added help text to ensure editors are aware
